README.md

```diff
@@ -106,15 +120,14 @@ Optional Description
 
 ### PRs
 
-Please title PR as 
+Please title PR as
 
 `type_of_branch: Main Description`
 
-Add optional Message to the body of the PR 
+Add optional Message to the body of the PR
 
 Once approved please squash PR, title and message should follow same convention as PR.
 
-
```

Comment:
It doesn't appear here:
-----------
## Docs

### Model

[Updated draw io Model](https://drive.google.com/file/d/10hqU88j-wviGUm4WkoZvQdNqPKNEO4dV/view?usp=sharing)

![Model](docs/ModeloDataBase_0_1.jpg)

-------------
and in the merge request exist some conflicts in this line with the last merge request